modules/howtos/pages/full-text-searching-with-sdk.adoc

```diff
@@ -2,81 +2,129 @@
 :navtitle: Searching from the SDK
 :page-topic-type: howto
 
-
 [abstract]
 You can use the Full Text Search service (FTS) to create queryable full-text indexes in Couchbase Server.
 
+Full Text Search (FTS) allows you to create, manage, and query full-text indexes on JSON documents stored in Couchbase buckets.
+
+Using natural language processing for querying documents, FTS provides relevance scoring on the results AKA 'hits' from your queries, and has fast indexes for querying a wide range of possible text searches.
 
-Full Text Search or FTS allows you to create, manage, and query full text indexes on JSON documents stored in Couchbase buckets.
-It uses natural language processing for querying documents, provides relevance scoring on the results of your queries, and has fast indexes for querying a wide range of possible text searches.
-Some of the supported query types include simple queries like Match and Term queries; range queries like Date Range and Numeric Range; and compound queries for conjunctions, disjunctions, and/or boolean queries.
-The Node.js SDK exposes an API for performing FTS queries which abstracts some of the complexity of using the underlying REST API.
+Supported query types include simple queries like Match and Term queries; range queries like Date Range and Numeric Range; and compound queries for conjunctions, disjunctions, and/or boolean queries.
 
+The Couchbase Node.js SDK exposes an API for performing FTS queries which abstract some of the complexity of using the underlying REST API.
 
 // As of Couchbase Server 6.5, FTS...
 
 == Examples
 
-Search queries are executed at the cluster level (not bucket or collection).
-Here is a simple MatchQuery that looks for the text “swanky” using a defined index:
+Search queries are executed at the cluster level (not bucket or collection). All of the examples below will log to the console our returned documents along with their metadata and rows, each returned document has an index, id, score and sort value.
```

Comment:
I assume this is against `travel-sample`?  We might want to say so.  We also might want to reference the 'getting started page for connection semantics, etc.